### ISSUE TITLE
Chezy Champs Double Elim Support

### ISSUE THREAD

**Issue by phil-lopreiato** [the-blue-alliance/the-blue-alliance#4716]
A few things we need to update from our old `DOUBLE_ELIM_8_TEAM` playoff format:

- match names should ideally be "round N" and not winner/loser bracket
- we need to build bracket support

![image](https://user-images.githubusercontent.com/2754863/189549469-eea5696f-2a75-4d34-9805-15eee4f6b5f8.png)

**Comment by phil-lopreiato** [comment]
I think this now means we're good to go!

**Comment by neyre** [comment]
Hi @phil-lopreiato @AndrewKe,

Thanks so much for getting this in on such short notice. Having the clear support on TBA for the bracket is so helpful from an event organizer perspective.

Right now, the bracket is listing "Unknown" instead of alliance names. Any way we can get this corrected? I realize it might be Cheesy Arena's fault, but if there's any way to just update the data on your end that'd be wonderful.

Thanks!

NIck

**Comment by phil-lopreiato** [comment]
Yeah; ack on this (I saw, but just didn't get the chance to fix it last night). Will fix!

**Comment by neyre** [comment]
Thanks, Phil!

Also, if y'all want the SVG files for the dynamically updated bracket, let me know, we could provide that if it's helpful for anything. (The Cheesy Arena bracket is just a SVG image that gets dynamically populated with the relevant data and then rendered in the browser).
